src/njoy21/input/LEAPR/Card17/Dka.hpp

```diff
@@ -0,0 +1,19 @@
+struct Dka {
+  using Value_t = decltype( pow( 1.0 * angstrom, Ratio<-1> ) );
```

Comment:
That's correct, dka has to be in units of inverse angstroms, and this was the cleanest was I could think of to show that. Again, maybe @apmccartney will swoop in with a more elegant statement